packages/brisa-tailwindcss/libs.json

```diff
@@ -0,0 +1,25 @@
+[
```

Comment:
This file is autogenerated each time that we are installing the dependencies of Brisa